### PR TITLE
add a more helpful token expired page

### PR DESCRIPTION
**Pull request by btkostner**
For the redirect loop, if you try using an expired token, you'll get this page instead of the unauthorized page. It will also clear any session data (log you off), so you can log in again and generate a new token instead of re-using an expired session token for login.

TL:DR / better explained:
This avoids the login redirect loop with an expired session token stored

### CODE REVIEW

File: lib/recognizer_web/controllers/fallback_controller.ex [hunk 1]
```diff
@@ -18,7 +18,16 @@ defmodule RecognizerWeb.FallbackController do
   end
 
   @impl Guardian.Plug.ErrorHandler
-  def auth_error(conn, {_type, _reason}, _) do
+  def auth_error(conn, {:invalid_token, _reason}, _) do
+    conn
+    |> Recognizer.Guardian.Plug.sign_out()
+    |> fetch_session()
+    |> clear_session()
+    |> respond(:unauthorized, "401-invalid")
```
Comment (doomspork) [inline lib/recognizer_web/controllers/fallback_controller.ex:26]: Why don't we redirect to login with a flash?

Comment (btkostner) [reply lib/recognizer_web/controllers/fallback_controller.ex:26]: Let me go back a couple of commits to trigger a login loop and test this. I don't think that would work, but I can't really describe why I think that, so might as well try it out. Give me a bit.

Comment (btkostner) [reply lib/recognizer_web/controllers/fallback_controller.ex]: Updated. Should work because we store the get params of the url.

File: lib/recognizer_web/templates/error/401-invalid.html.eex [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="box">
```
Comment (doomspork) [inline lib/recognizer_web/templates/error/401-invalid.html.eex:1]: Is this file still necessary?
